src/dmd/globals.d

```diff
@@ -266,6 +264,7 @@ extern (C++) struct Param
     // Linker stuff
     Array!(const(char)*) objfiles;
     Array!(const(char)*) linkswitches;
+    Array!bool linkswitchIsForCC;
```

Comment:
We could have also defined a struct of both properties and then just used a single array.

Reply:
Yes. Though, that would have involved touching all the other platforms' linker driver code. Can refactor in a follow-up.